src/encoded/static/components/experiment_series.js

```diff
@@ -520,10 +522,16 @@ class ExperimentSeriesComponent extends React.Component {
         if (Object.keys(viewableDatasets).length > 0) {
             // Add the "Add all to cart" button and internal tags from all related datasets.
             const experimentIds = Object.values(viewableDatasets).map(experiment => experiment['@id']);
+            const hubUrl = `http://genome.ucsc.edu/cgi-bin/hgTracks?hubClear=${window.location.origin}/batch_hub/type=Experiment%26accession=${
+                Object.values(viewableDatasets).map(experiment => experiment.accession).join('%26accession=')
+            }/hub.txt&db=hg38`;
             addAllToCartControl = (
                 <div className="experiment-table__header">
                     <h4 className="experiment-table__title">{`Experiments in experiment series ${context.accession}`}</h4>
-                    <CartAddAllElements elements={experimentIds} />
+                    <div className="experiment-table__header-controls">
+                        <CartAddAllElements elements={experimentIds} />
+                        <a className="btn btn-info btn-sm" href={hubUrl} type="button" target="_blank" rel="noopener noreferrer">Visualize</a>
```

Comment:
Did you decide the arrow after the “Visualize” label in the button is OK?

Reply:
Thanks for the help. I updated the code. Please take a look again.